modules/OCPP201/OCPP201.cpp

```diff
@@ -689,7 +690,14 @@ void OCPP201::ready() {
     callbacks.validate_network_profile_callback =
         [this](const int32_t configuration_slot,
                const ocpp::v201::NetworkConnectionProfile& network_connection_profile) {
```

Comment:
Currently, `configuration_slot` is not used, is it necessary at all?
Furthermore I would drop the captured `this`, because it is also not used.

Reply:
this will be relevant in the future for other validations